config/betterquesting/DefaultQuests/Quests/Tier12UMV-l37V2WvbSHGZQ5oMgAfBuw==/StoringComputati-lBuQcNYjQ8-YsMFqW6U1tQ==.json

```diff
@@ -0,0 +1,61 @@
+{
+  "questIDHigh:4": -7774461517421657137,
+  "preRequisites:9": {
+    "0:10": {
+      "questIDHigh:4": 0,
+      "questIDLow:4": 3244
+    }
+  },
+  "questIDLow:4": -7444237521495181899,
+  "properties:10": {
+    "betterquesting:10": {
+      "snd_complete:8": "random.levelup",
+      "taskLogic:8": "AND",
+      "visibility:8": "NORMAL",
+      "isMain:1": 0,
+      "simultaneous:1": 0,
+      "icon:10": {
+        "id:8": "gregtech:gt.blockmachines",
+        "Count:3": 1,
+        "Damage:2": 15444,
+        "OreDict:8": ""
+      },
+      "snd_update:8": "random.levelup",
+      "repeatTime:3": -1,
+      "globalShare:1": 0,
+      "questLogic:8": "AND",
+      "repeat_relative:1": 1,
+      "name:8": "Storing Computation in the Cloud",
+      "lockedProgress:1": 0,
+      "autoClaim:1": 0,
+      "isSilent:1": 0,
+      "desc:8": "Wireless Computation is similar to Wireless Data, Cloud Computation Server Hatches sends computation to Cloud Computation Client Hatches, which other multiblocks can use.\n\nPlace down a Server Hatch on a Quantum Computer, screwdriver the controller to enable Wireless Mode, then after placing down Client Hatches, you have to set how many computation it should request from the Server Hatch."
```

Comment:
```suggestion
      "desc:8": "Wireless Computation is similar to Wireless Data, Cloud Computation Server Hatches send computation to Cloud Computation Client Hatches, which other multiblocks can use.\n\nPlace down a Server Hatch on a Quantum Computer, screwdriver the controller to enable Wireless Mode, then after placing down Client Hatches, set how much computation it should request from the Server Hatch."
```

Reply:
May also be worth mentioning the 1:1 -> N:M relationship with these similar to the wireless data hatches